Meta/gn/secondary/Meta/Lagom/Tools/lagom_tool.gni

```diff
@@ -0,0 +1,16 @@
+template("lagom_tool") {
+  if (current_toolchain == host_toolchain) {
```

Comment:
This should only be used in lagom files, right? Maybe `assert(current_toolchain == host_toolchain)` instead?

Reply:
One day we might want to instal these tools (cross-compiled) onto the system so that the self-hosting story is a bit easier, but for now yeah, that's true. I'll assert host_toolchain.